core-ui/src/shared/components/GenericList/GenericList.scss

```diff
@@ -36,6 +36,7 @@
     }
   }
 
+  /*
```

Comment:
If this is not necessary then please remove it

Reply:
already did :)